spec/controllers/blog_controller_spec.rb

```diff
@@ -41,19 +41,20 @@
         let(:format) { nil }
 
         context 'when the requested file exists in the blog/ directory' do
-          let(:show_content) { 'it is the best language ever' }
+          let(:show_content) { '</head>it is the best language ever' }
 
           around do |example|
             with_blog_file("#{slug}.html", show_content) do
               example.run
             end
           end
 
-          it 'responds with 200 and the show file content' do
+          it 'responds with 200 and the show file content (with additions, such as a styles.css stylesheet tag)' do
             get_show
 
             expect(response).to have_http_status(200)
-            expect(response.body).to eq(show_content)
+            expect(response.body).to match(/\bstyles\b/)
```

Comment:
This test is slightly awkward / not tightly specified because, in prod, I think that this will be a stylesheet, but in specs I think it might be a script tag for `styles.ts`. So, I'm just checking for `styles`.